eng/pipelines/templates/pipeline.yml

```diff
@@ -84,6 +84,7 @@ resources:
 
 variables:
   DOTNET_NOLOGO: 1
+  CI: 'true'
```

Comment:
Was this not set before somewhere?  We have the CI-only facts and theories which is looking for this env var.  Now some tests are crashing on this PR build consistently.  I'm wondering if this change is making more tests run than before?

Reply:
It's set up as either a pipeline variable, or in our "variable group". I wanted to add it to the yaml just to be more explicit about which variables our CI depends on, but I really wasn't expecting any behaviour differences 😕